src/verilog.y

```diff
@@ -7198,8 +7197,7 @@ localNextId<nodeExprp>:         // local
         //                      // Must call nextId without any additional tokens following
                 yLOCAL__COLONCOLON
                         { $$ = new AstClassOrPackageRef{$1, "local::", PARSEP->unitPackage($<fl>1), nullptr};
-                          SYMP->nextId(PARSEP->rootp());
-                          BBUNSUP($<fl>1, "Unsupported: Randomize 'local::'"); }
+                          SYMP->nextId(PARSEP->rootp()); }
```

Comment:
local:: is valid to parse in many places, beyond just constraints and even currently in IEEE-illegal places (everywhere packageClassScope is used).   I don't see what will warn in all of those places when local:: is used with this warning removed.

Reply:
Added extra error.